companies/510057606.json

```diff
@@ -0,0 +1,20 @@
+{
+    "name": "Cariboulabs",
```

Comment:
where did you pull the ח.פ from?

Reply:
Cariboulabs is [516525334](https://ica.justice.gov.il/GenericCorporarionInfo/SearchCorporation?unit=8#)

Reply:
whoops. thank you!